engine/src/main/java/io/zeebe/engine/processing/streamprocessor/ReProcessingStateMachine.java

```diff
@@ -116,15 +116,15 @@
   private LoggedEvent currentEvent;
   private TypedRecordProcessor eventProcessor;
   private ZeebeDbTransaction zeebeDbTransaction;
-  private boolean detectReprocessingInconsistency;
+  private final boolean detectReprocessingInconsistency;
 
   public ReProcessingStateMachine(final ProcessingContext context) {
     actor = context.getActor();
     eventFilter = context.getEventFilter();
     logStreamReader = context.getLogStreamReader();
     recordValues = context.getRecordValues();
     recordProcessorMap = context.getRecordProcessorMap();
-    dbContext = context.getDbContext();
+    transactionContext = context.getDbContext();
```

Comment:
Rename transactionContext to dbContext or #getDbContext to getTransactionContext.